_data/accessibility-tests/link.yml

```diff
@@ -37,3 +37,6 @@ test_items:
     version_tested: 3.5.0
     wcag_criterion: 2.4.7
 # Screen reader items
+
+
+      
```

Comment:
**polish ** Looks like some additional lines were added to the link accessibility test page. Would you mind removing these?

```suggestion
```

Reply:
Resolved in [f359dbc](https://github.com/uswds/uswds-site/pull/2746/commits/f359dbc93ef16ed7a18a20dd41c5320ebc2bec63)